src/Units/Unit.java

```diff
@@ -46,27 +47,35 @@ public Unit(Commander co, UnitModel um)
 		if (model.holdingCapacity > 0)
 			heldUnits = new Vector<Unit>(model.holdingCapacity);
 	}
-	
-	public void initTurn()
-	{
+
+	public void initTurn(Location locus) {
 		isTurnOver = false;
 		fuel -= model.idleFuelBurn;
-		if (captureTarget != null && captureTarget.getResident() != this)
-		{
+		if (captureTarget != null && captureTarget.getResident() != this) {
 			captureTarget = null;
 			captureProgress = 0;
 		}
+		if (HP < 9.99) {
```

Comment:
Whoops, I commented on the specific commit, rather than the top-level diff, so let's add them again.

Why not use `HP < 10`? Or better, `HP < maxHP`, so we don't have magic numbers.